test/language/testdata/python/ParameterTypes.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+
```

Comment:
Could you please add an `import` statement for the `ParameterType` function as well? It's not strictly needed, but it helps maintainers understand where this function is defined.